### DISCUSSION TITLE
Hide namespace if translation is missing

### DISCUSSION THREAD

**Discussion by vutpov** [vinissimus/next-translate#806] [Q&A]
Hello, I want to display message coming from an API and if the message is in the configuration then it's all good, but if it is not it would show "common: Incorrect user or password!". Is there anyway I can hide the namespace?

**Comment by aralroca** [comment]
Maybe you can use fallbacks: https://github.com/vinissimus/next-translate#8-fallbacks

**Reply by vutpov** [reply]
@aralroca  That would just change it to the fallback. It would be nice if there is a config that I could set in i18n.js to not translate for missing translation, so the result would be "Incorrect user or password!" instead of "common: Incorrect user or password!".
